function/fetchFile.js

```diff
@@ -0,0 +1,56 @@
+/* Form of a Google Cloud Function URL (https://cloud.google.com/functions/docs/calling/http)
+   "https://YOUR_REGION-YOUR_PROJECT_ID.cloudfunctions.net/FUNCTION_NAME"
+*/
+
+
+var url = 'https://us-central1-sp19-codeu-35-7727.cloudfunctions.net/predict';
+var data = { } //An array within the displayImage implementation
+
+
+//Just prints the raw JSON data to the console
+fetch(url, {
+  method: 'POST', // or 'PUT'
+  body: data, // data can be `string` or {object}!
+  headers:{
+    'Content-Type': 'application/json'
+  }
+}).then(res => res.json())
+.then(response => console.log(JSON.stringify(response))) //Should return the image's predicted label (aka the breed) and the score
+.catch(error => console.error('Error:', error));
+
+
+
+//Handling the JSON body returned
+var myList = document.querySelector('ul'); //Will contain dog breed on one line and score one the next
+
+fetch(url, {
+  method: 'POST', // or 'PUT'
+  body: data, // data can be `string` or {object}!
+  headers:{
+    'Content-Type': 'application/json'
```

Comment:
This will need to be updated with the type of the file - detected from elsewhere.

e.g. in the example here - https://developers.google.com/web/fundamentals/media/capturing-images/#handling_a_filelist_object - the '.type` property of the file.

Reply:
Hi Matt! I was wondering, instead of putting an if statement in hi lighted code to double check the type of file, could we also just change the declaration of the image object?

Ex: <input type="file" accept="image/*">

Reply:
Correct - this is what we had in the previous version - and is what is in the Google Dev docs that I originally linked to.

Reply:
Okay! I currently have it as...

var data = { } //An array within the displayImage implementation

if (!data.type.match("image/*") {
    throw new Error("data is not of the type image");
}

Since data is established within https://github.com/codeu-spring2019/whats-that-doggo/blob/master/website/js/index.js , would it be best to combine that file and fetchFile.js ?